material-popup-menu/src/main/java/com/github/zawadz88/materialpopupmenu/MaterialPopupMenuBuilder.kt

```diff
@@ -20,14 +20,18 @@ import androidx.annotation.StringRes
 class MaterialPopupMenuBuilder {
 
     /**
-     * Style of the popup menu. Default is [R.style.Widget_MPM_Menu].
+     * Style of the popup menu.
      *
      * For dark themes you should use [R.style.Widget_MPM_Menu_Dark].
      *
-     * You can also provide your own style, however make sure that all of the attributes
-     * that are declared in [R.style.Widget_MPM_Menu] are also declared in your style.
+     * Setting this to `null` will make the popup use the default style resolved based on context
+     * passed to [MaterialPopupMenu.show] function. You can customize that default style by defining
+     * [R.attr.materialPopupMenuStyle] in your theme style.
+     *
+     * *NOTE:* make sure that all of the attributes that are declared in [R.style.Widget_MPM_Menu]
+     * are also declared in your style.
      */
-    var style: Int = R.style.Widget_MPM_Menu
+    var style: Int? = null
```

Comment:
I'm wondering if it wouldn't be better to use `var style: Int = 0` and check against 0 instead. I always find a bit icky to use `Int?` or a nullable `Integer` in Java. What do you think?

Reply:
Sounds good to me. I'll change that later.

Reply:
Cool, I'll review the rest of PR today/tomorrow. I would like to test this a bit more.

Reply:
Updated.